### PR TITLE
Fix access to CentOS 7 repositories after EOL

### PR DESCRIPTION
**Pull request by DavidePrincipi**
- Use vault.centos.org as YUM baseurl

Refs https://github.com/NethServer/dev/issues/6965

### CODE REVIEW

File: container/Containerfile [hunk 1]
```diff
@@ -1,4 +1,7 @@
 FROM centos:7 as makerpms
+RUN sed -i -e '/mirrorlist=http:\/\/mirrorlist.centos.org\// { s/^/#/ ; T }' \
+           -e '{ s/#baseurl=/baseurl=/ ; s/mirror\.centos\.org/vault.centos.org/ ; s/\$releasever/7.9.2009/ }' \
+           /etc/yum.repos.d/CentOS-*.repo
 RUN yum -y install \
```
Comment (Tbaile) [inline container/Containerfile]:
```suggestion
RUN sed -i -e '/mirrorlist=http:\/\/mirrorlist.centos.org\// { s/^/#/ ; T }' \
           -e '{ s/#baseurl=/baseurl=/ ; s/mirror\.centos\.org/vault.centos.org/ ; s/\$releasever/7.9.2009/ }' \
           /etc/yum.repos.d/CentOS-*.repo \
    && yum -y install \
```

**Review by gsanchietti — APPROVED** [review]
:crossed_fingers:

**Review by Tbaile — CHANGES_REQUESTED** [review]
This builds only the first `makerpms` image, the group `buildsys` and `devtoolset` are missing from the repo

**Review by Tbaile — APPROVED** [review]

**Review by Tbaile — CHANGES_REQUESTED** [review]
GitHub getting some fancy stuff lately with PRs

**Review by Tbaile — APPROVED** [review]

**Pull request merged**